hypothesis-python/tests/cover/test_integer_ranges.py

```diff
@@ -19,6 +19,8 @@
 
 import hypothesis.strategies as st
 from hypothesis import find, given, settings, unlimited
+from tests.common.utils import no_shrink
```

Comment:
Possibly daft question: where is this defined? I can’t see it.

Reply:
Just below the imports in that file - it was introduced when I switched most of our tests over, so it's not part of the diff for this PR.

Reply:
Huh, I thought I’d checked master for it as well – I swear it wasn’t there last night, but I’m apparently up-to-date…